calculator.js

```diff
@@ -0,0 +1,34 @@
+var jsCal = {
+  add: function(num1, num2){
+    return(num1 + num2);
+  },
+
+  sub: function(num1, num2){
+    return(num1 - num2);
+  },
+
+  multiply: function(num1, num2){
+    return(num1 * num2);
+  },
+
+  divid: function(bigNum, lilNum){
+    return(bigNum / lilNum);
+  },
+
+  expo: function(num1, num2){
+    var finalNum = 1;
+    for(i = 0; i<num2; i++){
+        finalNum = finalNum * num1;
+    }
+    return(finalNum);
```

Comment:
I hate to tell you this now, but JS actually has a built in exponent function: `Math.pow()`